apps/ecs/translations/src/en/fields.json

```diff
@@ -197,7 +197,7 @@
     "label": "Employer telephone number (optional)"
   },
   "before-1988-employer-email": {
-    "label": "Employer email address"
+    "label": "Employer email address (optional)"
```

Comment:
Change 'Employer email address (optional)' to 'Email address (optional)' to match figma